data/hulls/ship_data.csv

```diff
@@ -7,50 +7,50 @@ Arbitrator,diableavionics_arbitrator,,Diable Avionics,diableavionics_booster,,30
 Arbitrator Z,diableavionics_arbitratorZ,,Diable Avionics,,,350,75,400,,150,0,,200,150,75,90,180,10,NONE,,0,0,0,,,2,2,,,,,,,,,,,,,,,,,,,,100
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 Frost,diableavionics_frost,,Diable Avionics,diableavionics_slash,,800,100,500,,150,0,,200,250,250,90,180,50,NONE,,,,,,,3,3,,,,,,,,,,,,,,,,,,,,6
-Strife,diableavionics_strife,,Diable Avionics,diableavionics_dakka,,900,125,750,,200,0,,180,180,180,90,90,55,NONE,,,,,,,5,5,,,,,,,,,,,,,,,,,,,,8
-Hoar,diableavionics_hoar,,Diable Avionics,diableavionics_kata,,750,75,750,,200,0,,250,320,240,90,180,55,NONE,,,,,,,4,4,,,,,,,,,,,,,,,,,,,,8
+Strife,diableavionics_strife,,Diable Avionics,diableavionics_dakka,,1300,175,750,,200,0,,180,180,180,90,90,55,NONE,,,,,,,5,5,,,,,,,,,,,,,,,,,,,,8
+Hoar,diableavionics_hoar,,Diable Avionics,diableavionics_kata,,1000,75,750,,200,0,,250,320,240,90,180,55,NONE,,,,,,,4,4,,,,,,,,,,,,,,,,,,,,8
```

Comment:
Strife hull increased to 1300 (from 900), armor increased to 175 (from 125)
Hoar hull increased to 1000 (from 750)